integration/spark/build.gradle

```diff
@@ -128,25 +153,27 @@ test {
     }
     systemProperties = [
             'junit.platform.output.capture.stdout': 'true',
-            'junit.platform.output.capture.stderr': 'true'
+            'junit.platform.output.capture.stderr': 'true',
+            'spark.version'                       : project.getProperty('spark.version'),
+            'openlineage.spark.jar': "${archivesBaseName}-${project.version}.jar"
     ]
+    classpath =  project.sourceSets.test.runtimeClasspath + (isSpark3 ? configurations.spark3Test : configurations.spark2Test)
```

Comment:
If you update the configurations to extend `testImplementation`, you don't have to do this concatenation. E.g., see the docs at https://docs.gradle.org/current/userguide/declaring_dependencies.html:

```groovy
configurations {
    smokeTest.extendsFrom testImplementation
}
```

Reply:
I guess it's more complex. It was not working for me.
Receiving a lot of ClassNotFoundException
`Caused by: java.lang.ClassNotFoundException: com.google.cloud.spark.bigquery.repackaged.com.google.cloud.bigquery.MockBigQueryRelationProvider$MockInjector
...
Caused by: java.lang.ClassNotFoundException: io.openlineage.spark.agent.lifecycle.plan.JDBCRelationVisitorTest
...
`
For 
```
spark3Test.extendsFrom testRuntimeClasspath
spark2Test.extendsFrom testRuntimeClasspath
```
and
```
spark3Test.extendsFrom testImplementation
spark2Test.extendsFrom testImplementation
```

and usage
`classpath = isSpark3 ? configurations.spark3Test : configurations.spark2Test`

`classpath = configurations.spark3.runtimeClasspath` fails with
> Could not get unknown property 'runtimeClasspath' for configuration ':spark2Test'